src/test/java/chess/domain/position/ChessRankTest.java

```diff
@@ -0,0 +1,52 @@
+package chess.domain.position;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class ChessRankTest {
+
+    @DisplayName("주어진 Rank의 값으로 Rank을 찾는다.")
+    @Test
+    void findByValue() {
+        // given
+        String rankValue = "1";
+
+        // when
+        ChessRank rank = ChessRank.findByValue(rankValue);
+
+        // then
+        assertThat(rank).isEqualTo(ChessRank.ONE);
+    }
+
+    @DisplayName("체스 랭크 범위에 해당하지 않는 값을 조회하면 예외를 발생시킨다.")
+    @Test
+    void unknownChessRankValue() {
+        // given
+        String rankValue = "0";
+
+        // when & then
+        assertThatThrownBy(() -> ChessRank.findByValue(rankValue))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("체스 랭크 범위에 해당하지 않는 값입니다.");
+    }
+
+    @DisplayName("랭크와 랭크 사이의 랭크들을 조회한다.")
+    @Test
+    void findRankBetween() {
+        // given
+        ChessRank start = ChessRank.ONE;
+        ChessRank end = ChessRank.EIGHT;
+        List<ChessRank> expected = List.of(ChessRank.TWO, ChessRank.THREE, ChessRank.FOUR, ChessRank.FIVE, ChessRank.SIX, ChessRank.SEVEN);
```

Comment:
라인이 긴 것 같은데 아래처럼 개행하는건 어떨까요?
```suggestion
        List<ChessRank> expected = List.of(
            ChessRank.TWO, 
            ChessRank.THREE, 
            ChessRank.FOUR, 
            ChessRank.FIVE, 
            ChessRank.SIX, 
            ChessRank.SEVEN
        );
```

Reply:
반영해보겠습니다:)

혹시 실무 코드를 작성하실 때도 이렇게 한 개씩 개행하는 방식이 많이 사용되나요??
이렇게 개행해본 적이 없어서, 앞으로도 비슷한 코드에서 이렇게 작성해도 괜찮을 지 궁금해서 질문드렸습니다!

Reply:
저는 이럴땐 요소 한개마다 개행을 하고 있습니다!
뭐뭐가 들어가는지, 몇개나 들어가는지 한눈에 보기 편하려구요!

Reply:
> 저는 이럴땐 요소 한개마다 개행을 하고 있습니다! 뭐뭐가 들어가는지, 몇개나 들어가는지 한눈에 보기 편하려구요!

저도 알렉스 방식을 따라가야겠어요 ㅋㅋㅋㅋ
한 눈에 들어오는게 너무 편하네요:) 좋은 팁 배웠습니다!